### ISSUE TITLE
[zti] cursor disappears when on an input field with underscores

### ISSUE THREAD

**Issue by Christopher-Chappell** [IBM/tnz#77]
For example, from ISPF 3.4 you might see the following screen:
<img width="652" alt="image" src="https://github.com/IBM/tnz/assets/56929130/12cf64c6-a88c-4caa-a842-0ff62f53c577">

From zti, if the cursor is on top of one of the input fields, it is essentially invisible (at least with macos Terminal).  I can turn on insert mode in order to see the cursor, or move it, but for some applications you need to turn on overtype mode in order to properly interact with those fields.  I don't know if there is any way to make the cursor appear in these circumstances.  I vaguely remember there being a crosshair option which may or may not be useful, but ideally I shouldn't need that to be able to see my cursor's position.  Changing the cursor shape in Terminal settings also doesn't affect the appearance when in session, so I'm guessing that is to some extent controlled by zti.

**Comment by najohnsn** [comment]
Zti is trying to set the "blinking underline" cursor setting. Unfortunately, your terminal emulator does not seem to work well with such a cursor when it is on an underlined space. Maybe, instead, the "default" cursor should be used? Though the "blinking underline" was chosen so that it would be the same on all emulators - as opposed to letting the emulator choose what the "default cursor" should look like.

**Comment by Christopher-Chappell** [comment]
Yeah, even if it was just something that lets me toggle between ZTI's setting and my default.

**Comment by AnthonyGiorgio** [comment]
For what it's worth, the GNOME Terminal application properly supports the blinking underline cursor.  Here's how it looks on my system.
![image](https://github.com/IBM/tnz/assets/17408915/a6154812-743a-48ee-96e9-76fbcd53da91)

**Comment by najohnsn** [comment]
After seeing the above screenshot, it made me realize that colors look _off_ in the original screenshot. I'm not confident, but maybe if the colors get fixed, the cursor will become visible? @Christopher-Chappell, can you try  `export TNZ_COLORS=256` before running `zti` and see if that changes your experience? (at the very least, it should fix your colors)

**Comment by Christopher-Chappell** [comment]
that might just be how I set up my color settings, but I'll try it

**Comment by Christopher-Chappell** [comment]
<img width="689" alt="image" src="https://github.com/IBM/tnz/assets/56929130/f1b86701-2ff1-4390-9ba3-7a0e4f8f0dd4">
well the colors are fixed, but I still don't see the underscore appearing.  I'm guessing this is a Terminal.app thing.

**Comment by najohnsn** [comment]
Yeah, it's a Terminal.app thing it doesn't provide a way to see an _underline cursor_ when it is on top of an underline character.

Your colors still surprise me a bit. Notice how Anthony's "menu bar" is in white letters and yours is in blue. But I suppose that could be your ISPF color settings. Maybe you have White set to BLUE or something instead of this:
![image](https://github.com/IBM/tnz/assets/26844614/ffb17b04-7477-40d6-b801-0c73825b91ef)

**Comment by Christopher-Chappell** [comment]
The white text menu bar might be related to the "tab to action bar choices" option in ISPF options.  Turning it off changes the color of those items.

I while back I went through on our development system and tried a few different color arrangements, switching blue / green / turquoise / white around to where I found the combination less jarring, and drew my attention to fields more easily.  I eventually settled on the setting I have now, but I realize it's not default.

**Comment by najohnsn** [comment]
@Christopher-Chappell, in cases where you see the cursor in zti, is it blinking?

**Comment by Christopher-Chappell** [comment]
> @Christopher-Chappell, in cases where you see the cursor in zti, is it blinking?

Yeah, it does blink after a couple seconds when the cursor is on top of any other character.  I tried squinting at my screen when it's on top of an underline and it does not blink or change anything there.

**Comment by v1gnesh** [comment]
Just seeing this.. I get this too, when in 3.17.
Kinda unrelated but when in Omeg's E3270UI, if I do a `/`, I don't get the pop-up window.
This is with Windows Terminal and cmd.exe
